testfiles/SurfacePropTest_SurfLWR.idf

```diff
@@ -2285,12 +2281,15 @@
     SrdSurfs:Zn005:Wall002:Win001,  !- Name
     0.5,                     !- Sky View Factor
     ,                        !- Sky Temperature Schedule Name
-    0.3,                     !- Ground View Factor
-    ,                        !- Ground Temperature Schedule Name
     SurroundingSurface1,     !- Surrounding Surface 1 Name
     0.2,                     !- Surrounding Surface 1 View Factor
     Surrounding Temp Sch 2;  !- Surrounding Surface 1 Temperature Schedule Name
 
+ SurfaceProperty:GroundSurfaces,
+    GndSurfs:Zn005:Wall002:Win001,  !- Name
+    Gnd Surfs Wall002:Win001 Name,  !- Ground Surface 1 Name
+    0.3;                     !- Ground Surface 1 View Factor
+
```

Comment:
Added new object to replace removed fields for an existing example idf. This is manual transition.